core/tests/loadnext/src/sdk/ethereum/mod.rs

```diff
@@ -185,7 +213,6 @@ impl<S: EthereumSigner> EthereumProvider<S> {
         l1_token_address: Address,
         bridge: Option<Address>,
     ) -> Result<Address, ClientError> {
-        // TODO(EVM-571): This should be moved to the shared bridge, which does not have `l2_token_address` on L1. Use L2 contracts instead.
         let bridge = bridge.unwrap_or(self.default_bridges.l1_erc20_default_bridge.unwrap());
```

Comment:
Would it be possible to go via the L2SharedBridge? That would be a better long term solution. The L1 erc20 bridge is deprecated, we should avoid using it

Reply:
we don't use this method anymore, I can remove it